source/server/shared_library.h

```diff
@@ -16,16 +18,18 @@ typedef HMODULE LibraryHandle;
 typedef void* LibraryHandle;
 #endif
 
-struct LibraryLoadException : std::runtime_error {
```

Comment:
Note to reviewers: since `std::runtime_error` is a class, it feels really wrong to me for a struct to inherit from a class.